src/matchers/run-query.js

```diff
@@ -11,32 +11,37 @@
  */
 /* eslint-disable camelcase */
 const { fetch } = require('@adobe/helix-fetch');
+const { utils } = require('@adobe/helix-shared');
 
 async function extract(url, params, log = console) {
   const host = 'https://adobeioruntime.net';
-  const path = '/api/v1/web/helix/helix-services/run-query@latest/';
+  const path = '/api/v1/web/helix/helix-services/run-query@2.4.11/';
```

Comment:
```suggestion
  const path = '/api/v1/web/helix/helix-services/run-query@v2/';
```

You don't want to pin the exact version, otherwise you won't get new queries and bug fixes.